bundle/Templating/Twig/Extension/SortClauseExtension.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Netgen\Bundle\EzPlatformSiteApiBundle\Templating\Twig\Extension;
+
+use Netgen\Bundle\EzPlatformSiteApiBundle\Search\SortClauseParser;
+use Twig_Extension;
+use Twig_SimpleFunction;
+
+/**
+ * todo
+ */
+class SortClauseExtension extends Twig_Extension
+{
+    /**
+     * @var \Netgen\Bundle\EzPlatformSiteApiBundle\Search\SortClauseParser
+     */
+    private $sortClauseParser;
+
+    /**
+     * @param \Netgen\Bundle\EzPlatformSiteApiBundle\Search\SortClauseParser
+     */
+    public function __construct(SortClauseParser $sortClauseParser)
+    {
+        $this->sortClauseParser = $sortClauseParser;
+    }
+
+    public function getName()
```

Comment:
No need for this one. Twig does not use this since ages.

Reply:
Removed in https://github.com/netgen/ezplatform-site-api/pull/70/commits/9a214e971b5ca0c6e8c28b39d9ea0df87a8a1fad